### PR TITLE
Tweak library query

### PR DESCRIPTION
**Pull request by AntsyLich**
Output wise last read, latest chapter upload date and chapter fetch date are included alongside the old outputs. This data were previously collected via seperate queries.

Overall we get the data from a single compact query

The query is modified slightly after extracting from #5734

### CODE REVIEW

**Comment by arkon** [comment]
Can you put more details about what the changes are in the PR description?

File: app/src/main/java/eu/kanade/data/manga/MangaMapper.kt [hunk 1]
```diff
@@ -55,5 +55,8 @@ val libraryManga: (Long, Long, String, String?, String?, String?, List<String>?,
             category = category,
             unreadCount = unreadCount,
             readCount = readCount,
+            latestUpload = latestUpload,
+            chapterFecthedAt = chapterFecthedAt,
```
Comment (arkon) [inline app/src/main/java/eu/kanade/data/manga/MangaMapper.kt:59]: Fetched

File: app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryPresenter.kt [hunk 2]
```diff
@@ -394,16 +360,10 @@ class LibraryPresenter(
...
-                    val manga2latestChapter = latestChapterManga[i2.libraryManga.manga.id]
-                        ?: latestChapterManga.size
-                    manga1latestChapter.compareTo(manga2latestChapter)
+                    i1.libraryManga.latestUpload.compareTo(i2.libraryManga.latestUpload)
                 }
                 LibrarySort.Type.ChapterFetchDate -> {
-                    val manga1chapterFetchDate = chapterFetchDateManga[i1.libraryManga.manga.id] ?: 0
-                    val manga2chapterFetchDate = chapterFetchDateManga[i2.libraryManga.manga.id] ?: 0
-                    manga1chapterFetchDate.compareTo(manga2chapterFetchDate)
+                    i1.libraryManga.chapterFecthedAt.compareTo(i2.libraryManga.chapterFecthedAt)
```
Comment (AntsyLich) [inline app/src/main/java/eu/kanade/tachiyomi/ui/library/LibraryPresenter.kt:366]:
```suggestion
                    i1.libraryManga.chapterFetchedAt.compareTo(i2.libraryManga.chapterFetchedAt)
```

File: app/src/main/java/eu/kanade/data/manga/MangaMapper.kt (hunk 1 above)
Comment (AntsyLich) [inline app/src/main/java/eu/kanade/data/manga/MangaMapper.kt:59]:
```suggestion
            chapterFetchedAt = chapterFetchedAt,
```

File: app/src/main/java/eu/kanade/data/manga/MangaMapper.kt [hunk 3]
```diff
@@ -28,11 +28,11 @@ val mangaMapper: (Long, Long, String, String?, String?, String?, List<String>?,
         )
     }
 
-val libraryManga: (Long, Long, String, String?, String?, String?, List<String>?, String, Long, String?, Boolean, Long?, Long?, Boolean, Long, Long, Long, Long, UpdateStrategy, Long, Long, Long) -> LibraryManga =
-    { _id, source, url, artist, author, description, genre, title, status, thumbnailUrl, favorite, lastUpdate, nextUpdate, initialized, viewerFlags, chapterFlags, coverLastModified, dateAdded, updateStrategy, unreadCount, readCount, category ->
+val libraryManga: (Long, Long, String, String?, String?, String?, List<String>?, String, Long, String?, Boolean, Long?, Long?, Boolean, Long, Long, Long, Long, UpdateStrategy, Long, Long, Long, Long, Long, Long) -> LibraryManga =
+    { id, source, url, artist, author, description, genre, title, status, thumbnailUrl, favorite, lastUpdate, nextUpdate, initialized, viewerFlags, chapterFlags, coverLastModified, dateAdded, updateStrategy, unreadCount, readCount, latestUpload, chapterFecthedAt, lastRead, category  ->
```
Comment (AntsyLich) [inline app/src/main/java/eu/kanade/data/manga/MangaMapper.kt:32]:
```suggestion
    { id, source, url, artist, author, description, genre, title, status, thumbnailUrl, favorite, lastUpdate, nextUpdate, initialized, viewerFlags, chapterFlags, coverLastModified, dateAdded, updateStrategy, unreadCount, readCount, latestUpload, chapterFetchedAt, lastRead, category ->
```

**Comment by arkon** [comment]
Should bump the version code to prevent app downgrades too if you're making database schema changes.

**Pull request merged**